tensorflow/python/ops/math_ops.py

```diff
@@ -4237,6 +4237,25 @@ def polyval(coeffs, x, name=None):
   Returns:
     A `tensor` of the shape as the expression p(x) with usual broadcasting
     rules for element-wise addition and multiplication applied.
+    
```

Comment:
Please remove trailing whitespace.

Reply:
Thanks for reviewing! I've made the changes.